packages/donor/src/screens/questionnaire/QuestionnaireScreen.tsx

```diff
@@ -236,50 +126,20 @@ export default function QuestionnaireScreen({
           if (hasAlreadyDonated === "no") {
             setHasAlreadyDonated("");
           }
-          if (isWeightOver55 === "no") {
-            setIsWeightOver55("");
+          if (isWeightValid === "no") {
```

Comment:
Why are we comparing to a string and not bool or enum here?

Reply:
I am not sure, but I didn't change this.
We can change this in a different PR